src/main/java/com/artipie/front/auth/YamlCredentials.java

```diff
@@ -21,22 +32,57 @@
 public final class YamlCredentials implements Credentials {
 
     /**
-     * Yaml source.
+     * Cache for credentials settings.
      */
-    private final YamlMapping mapping;
+    private final LoadingCache<String, YamlMapping> creds;
+
+    /**
+     * Ctor.
+     * @param creds Credentials
+     */
+    public YamlCredentials(final LoadingCache<String, YamlMapping> creds) {
```

Comment:
We cannot use any `LoadingCache` here, only with a certain `CacheLoader` that ignores a passed key.
Need to change logic or remove this constructor.